ansible-scylla-node/tasks/upgrade/main.yml

```diff
@@ -0,0 +1,305 @@
+---
```

Comment:
can we enforce serial: 1 directly here?
there is no way we will be running upgrade on multiple nodes at once

Reply:
AFAIK, you can't using inside a role since it's a playbook attribute. Here's the official doc: https://docs.ansible.com/ansible/latest/user_guide/playbooks_strategies.html